### PR TITLE
fix(sync_replay): fix replay and insert tps to report

### PR DESCRIPTION
**Pull request by chanhsu001**
- fix replay action failure and grep replay tps from log
 - insert replay_tps into report
 - shfmt on sync-mainnet.sh (please do shell format to other shell scripts)

i've test the changes in test branch.
Please help to **modify db table structure**(add  `replay_tps` column) as `schema.sql`, @liya2017

### CODE REVIEW

File: ckb-sync-mainnet/script/schema.sql [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-	speed BIGINT NOT NULL,
-	tip BIGINT NOT NULL,
-	hostname VARCHAR (60) NOT NULL
+    ckb_commit_id VARCHAR (20) NOT NULL,
+    ckb_commit_time TIMESTAMP NOT NULL,
+    time_s TIME NOT NULL,
+    speed BIGINT NOT NULL,
+    tip BIGINT NOT NULL,
+    hostname VARCHAR (60) NOT NULL,
+    replay_tps INT,
```
Comment (liya2017) [inline ckb-sync-mainnet/script/schema.sql:24]:
```suggestion
    replay_tps INT
```

File: ckb-sync-mainnet/script/sync-mainnet.sh [hunk 2]
```diff
@@ -16,7 +16,10 @@ GITHUB_TOKEN=${GITHUB_TOKEN}
 
 JOB_ID=${JOB_ID:-"sync-mainnet-$(date +'%Y-%m-%d')-in-10h"}
 TAR_FILENAME="ckb.sync-mainnet-$(date +'%Y-%m-%d')-in-10h.tar.gz"
-SCRIPT_PATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+SCRIPT_PATH="$(
+  cd -- "$(dirname "$0")" >/dev/null 2>&1
+  pwd -P
```
Comment (liya2017) [inline ckb-sync-mainnet/script/sync-mainnet.sh:21]:
```suggestion
```

**Review by liya2017 — CHANGES_REQUESTED** [review]

File: ckb-sync-mainnet/script/sync-mainnet.sh (hunk 2 above)
Comment (chanhsu001) [reply ckb-sync-mainnet/script/sync-mainnet.sh]: i revert to `SCRIPT_PATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"` cause  i cannot sure if shell reformat is ok

**Comment by chanhsu001** [comment]
i revert to SCRIPT_PATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )" cause i cannot sure if shell reformat is ok

**Comment by chanhsu001** [comment]
test result: https://github.com/nervosnetwork/ckb/runs/5966847003?check_suite_focus=true
start process failed cause of timeout(15 minutes set) and replay get TPS(325)
```
ok: [instance-us-west-1] => {
    "msg": "325"
}
```
but I cannot check if  final data  all insert into DB (running log cannot show it)
please help to double check. @liya2017

File: ckb-sync-mainnet/script/schema.sql [hunk 3]
```diff
@@ -1,27 +1,27 @@
 CREATE DATABASE ckbtest;
-\c ckbtest 
 CREATE EXTENSION IF NOT EXISTS timescaledb CASCADE;
 
 CREATE TABLE IF NOT EXISTS sync_mainnet (
     github_run_id INT NOT NULL,
```
Comment (liya2017) [inline ckb-sync-mainnet/script/schema.sql:5]:
```suggestion
    github_run_id FLOAT NOT NULL,
```

Comment (doitian) [reply ckb-sync-mainnet/script/schema.sql]: Bigint is better

File: ckb-sync-mainnet/script/schema.sql [hunk 4]
```diff
@@ -1,27 +1,27 @@
 CREATE DATABASE ckbtest;
-\c ckbtest 
 CREATE EXTENSION IF NOT EXISTS timescaledb CASCADE;
 
 CREATE TABLE IF NOT EXISTS sync_mainnet (
-    github_run_id INT NOT NULL,
+    github_run_id FLOAT NOT NULL,
```
Comment (liya2017) [inline ckb-sync-mainnet/script/schema.sql:5]:
```suggestion
    github_run_id BIGINT NOT NULL,
```

**Review by doitian — APPROVED** [review]

**Review by liya2017 — APPROVED** [review]

**Comment by chanhsu001** [comment]
bors r=liya2017,doitian

**Comment by doitian** [comment]
> bors r=liya2017,doitian

no bors here 😂

**Pull request merged**